dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/OmegaRef/CMakeLists.txt

```diff
@@ -25,9 +25,11 @@ set(MODEL_FILES
   GeneratorSynchronous.extvar
   GeneratorSynchronousInt_INIT.mo
   GeneratorSynchronousExt_4E_INIT.mo
+  GeneratorSynchronousExt_4E_i0Non0_INIT.mo
```

Comment:
I prefer the modification of the existing _INIT model. Also, could you check that a start = 0 is equivalent for the other nrt to not putting a start ?

Reply:
I have modified the existing _INIT.mo by adding a parameter iStart0Pu (default value = Complex(0,0)) and deleted the models I had created. The default zero start value for i0Pu leads to successful nrt in all cases, in dynawo and dynawo-rte.

This start parameter iStart0Pu is modified in the test cases involving VRSemipol2, allowing the simulation to run.

This PR can thus be merged.